### DISCUSSION TITLE
Colors map

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1908] [Q&A]
Hello
How can I change the colors of the map?
I require them darker.

![image](https://user-images.githubusercontent.com/117297044/210258110-e0b09df1-38ab-48d0-acac-a12eb1698535.png)

Thank you

**Comment by sarah-beasley** [comment]
Hi, we are working on updating the code so that all users can update the map colours to any of their choosing in our next release of Open SDG - 2.2.0. We'll give further instructions when this has been released so you can change the colours as you wish. We would advise not to go too dark though, otherwise the map labels might not be readable and could make the maps less accessible! :) 

Although not perfect, the colours we use on our UK maps have been tested to the most accessible they can be, and they are like this FYI: 
![image](https://user-images.githubusercontent.com/106670682/210342753-b8e88235-9c65-482e-b472-56fb6c211712.png)
#c4e1c6
#b0d1b3
#9bc2a1
#87b28f
#74a37c
#60946b
#4d8559
#3a7747
#276836

And the following colours are used for the map backgrounds:
Text to (Black), #000000)
Turn off the text halo
Water to (Grey) #949494
Land to (white), #FFFFFF

**Reply by SandraEliz** [reply]
Hello
And those colors in which file do you add them?

**Comment by christoph-LSN** [comment]
@SandraEliz  @sarah-beasley 

Perhaps you can try this:

https://github.com/christoph-LSN/MT_Site/blob/a4861e125e665585104f53f37e1ede54c42f4bcf/_data/site_config.yml#L218

Docs: https://gka.github.io/chroma.js/

example https://christoph-lsn.github.io/MT_Site/1-1-1/

![image](https://user-images.githubusercontent.com/94443931/210353261-8a3d0b4b-77bb-46f2-893c-ab31595f9971.png)

**Comment by brockfanning** [comment]
@SandraEliz As mentioned, we're working on different ways to control the color range. At the moment, the way to control the color range is in the `colorRange` property in the `map_options` site configuration setting. So, this would typically be done in the _data/site_config.yml file. You should choose one of the available `chroma.brewer` color ranges. To see other available color ranges, see here: https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3

Try clicking some other swatches under "Pick a color scheme". When you pick a swatch, you will see its code appear below after "3-class":

![image](https://user-images.githubusercontent.com/1319083/212488828-4d9fabc0-80e2-42dd-a463-ff2434f37325.png)

You can use that code in the colorRange setting, after `chroma.brewer.`. For example, to use the `YlOrRd` range, you would add `chroma.brewer.YlOrRd`.

Again, as mentioned, we hope to make it easier to have completely custom color ranges in the next release of Open SDG.

**Reply by SandraEliz** [reply]
Thanks
